### DISCUSSION TITLE
GPU programming with cuda/opencl

### DISCUSSION THREAD

**Discussion by lenzi-e** [SLO-Coders-Group/slo-coders#70] [Shop Talk ideas]
open discussion about GPU programming with cuda/opencl

**Comment by cannadayr** [comment]
similar things I'm interested in:
GPGPU programming model (potentially contrasted with SIMD).
Shaders (SPIR-V, Slang)
GPU DSL's (Futhark)

**Reply by cannadayr** [reply]
or just general "what is SIMT, what can you do with gpgpu, and how do you structure programs?"
